### ISSUE TITLE
what is $PU?

### ISSUE THREAD

**Issue by kdaily** [bsmn/bsmn-pipeline#21]
https://github.com/bsmn/bsmn-pipeline/blob/master/genome_mapping/job_scripts/aln_1.align_sort.sh#L13

**Comment by kdaily** [comment]
I think it's 'platform unit'. Is it from the fastq header composed of the `{FLOWCELL_BARCODE}.{LANE}.{SAMPLE_BARCODE}`?

**Comment by bintriz** [comment]
You're right. It stands "platform unit" and is extracted from each read name in the fastq while running genome_mapping/job_scripts/pre_2.split_fastq_by_RG.sh. The format here I'm using is "FCX"_L"LN".
